src/components/Pages/Members/Members.jsx

```diff
@@ -194,3 +197,32 @@ export class Members extends React.PureComponent {
     );
   }
 }
+
+const mapStateToProps = (state) => {
+  return {
+    users: state.users.users,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    getUsers: () => dispatch(getUsersThunk()),
+    removeUser: (userId) => dispatch(removeUserThunk(userId)),
+    editUser: (userId, userData) => dispatch(editUserThunk(userId, userData)),
+    createUser: (userData) => dispatch(createUserThunk(userData)),
```

Comment:
use `bindActionCreators` to make code more concise

Reply:
you can write like `bindActionCreators({removeUser: removeUserThunk, editUser: editUserThunk}, dispatch)` as far as I remember